test/core/compression/args_utils.cc

```diff
@@ -25,6 +25,15 @@ const grpc_channel_args*
 grpc_channel_args_set_channel_default_compression_algorithm(
     const grpc_channel_args* a, grpc_compression_algorithm algorithm) {
   GPR_ASSERT(algorithm < GRPC_COMPRESS_ALGORITHMS_COUNT);
+  if (a != nullptr) {
+    for (size_t i = 0; i < a->num_args; i++) {
```

Comment:
This could use `grpc_channel_args_find()` instead.

Reply:
ended up not needing to do this!